stream/build.gradle.kts

```diff
@@ -52,4 +52,5 @@ testModuleInfo {
     requires("com.google.protobuf.util")
     requires("org.junit.jupiter.api")
     requires("org.junit.jupiter.params")
+    requiresStatic("com.github.spotbugs.annotations")
```

Comment:
```suggestion
```

I think this addition is not necessary. The dependency check might even fail now on this (I rebased this PR to check).